### PR TITLE
WRP-2673: Add cookie policy banner and page

### PR DESCRIPTION
**Pull request by sjro**
Add cookie policy banner and cookie policy description page.

### CODE REVIEW

File: src/components/SiteFooter/SiteFooter.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // SiteFooter
 //
+
```
Comment (juwonjeong) [inline src/components/SiteFooter/SiteFooter.js:3]: Please remove this line.

File: src/html.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from "react"
```
Comment (juwonjeong) [inline src/html.js:1]: I think the indents in this file are using spaces. Please convert it to Tabs with size 4.

File: src/components/SiteFooter/SiteFooter.js (hunk 1 above)
Comment (sjro) [reply src/components/SiteFooter/SiteFooter.js]: fixed it

File: src/html.js (hunk 2 above)
Comment (sjro) [reply src/html.js:1]: fixed it

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**